src/utils/abi-decoder.ts

```diff
@@ -112,18 +112,21 @@ export default class ABIDecoder {
         // const abiItem = await this.getABIFunctionFromExternalSource(methodID)
         const abiResult = this.methodSigs[methodID] || (await this.db.getFirstABIForHexSignature(methodID))
         const abiItem = abiResult ? ABI_FunctionZ.parse(abiResult) : null
-
         if (abiItem) {
-            const decoded = abiCoder.decodeParameters(abiItem.inputs, data.slice(10))
+            let decoded = {};
```

Comment:
nit `let decoded: Record<any, any> = {}` so you don't have to cast it to `any` on L129